webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala

```diff
@@ -174,57 +173,16 @@ final class AsyncDeploymentActor(
     )
   }
 
-  // That flag file is used to determine if a policy update was running when the webapp stopped so we can launch a new one when starting
-  val policyUpdateRunningFlagPath = "/opt/rudder/etc/policy-update-running"
```

Comment:
Shouldn't we delete that one during `rudder-upgrade`?

Reply:
good point

Reply:
done